plugins/chain_plugin/include/eosio/chain_plugin/trx_retry_db.hpp

```diff
@@ -2,12 +2,10 @@
 #include <eosio/chain/types.hpp>
 #include <eosio/chain/block_state.hpp>
 #include <eosio/chain/trace.hpp>
+#include <eosio/chain/transaction_metadata.hpp>
```

Comment:
Do you still need this include?

Reply:
Good catch, you are right we don't need it, I have removed it.